config/environments/dev.rb

```diff
@@ -75,6 +75,18 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = true
+
+  config.action_mailer.smtp_settings = {
+    domain: ENV.fetch('HOST'),
+    address: ENV.fetch('SMTP_SERVER'),
+    port: ENV.fetch('SMTP_PORT', 587).to_i
+  }
+
+  config.action_mailer.default_url_options = { host: ENV.fetch('HOST', 'challenge.gov') }
```

Comment:
not a blocker but I suppose this _could_ default https://challenge-dev.app.cloud.gov/
I checked and the HOST var already has that value anyways.
```suggestion
  config.action_mailer.default_url_options = { host: ENV.fetch('HOST', 'challenge-dev.app.cloud.gov') }
```